src/settings/VoucherNumber/SettingsVoucherNumberForm.js

```diff
@@ -27,12 +27,12 @@ const SettingsVoucherNumberForm = ({ onReset, firstSequenceNumber }) => (
     </Row>
     <Row>
       <Col xs={12}>
-        <Field
-          component={TextField}
+        <TextField
           label={<FormattedMessage id="ui-invoice.settings.voucherNumber.startNumber" />}
           name="sequenceNumber"
-          disabled
-          required
+          type="number"
+          value={1}
```

Comment:
this makes not too much sense to me. Could you please elaborate more on this?

Reply:
This used like default value.

Reply:
you have too many magic numbers in this PR. I've saw at least 2 places with `1`.

Reply:
make a constant at least.

Reply:
fixed